.github/workflows/ci-push.yml

```diff
@@ -226,7 +226,26 @@ jobs:
           raco pkg config --set catalogs $PWD/rktcat/ https://pkgs.racket-lang.org https://planet-compats.racket-lang.org
           raco pkg install --auto racket-test
       - name: Run tests
-        run: raco test -l tests/racket/test
+        run: |
```

Comment:
One drawback of writing it this way is that it gets run as a single shell command, so it's harder to tell what failed.

Reply:
Right - I will make it one step per test before merging.